### PR TITLE
Generate data for Network API according to NuPIC #2339

### PR DESCRIPTION
**Pull request by BoltzmannBrain**

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@scottpurdy please review.

**Comment by BoltzmannBrain** [comment]
Run w/ e.g. `python fluent/utils/network_data_generator.py -fin data/sample_reviews_multi/sample_reviews_data_training.csv`

File: data/sample_reviews_multi/sample_reviews_data_training_network.csv [hunk 1]
```diff
@@ -1 +1,503 @@
...
"lecture series; guest lectures, videos, or we give them oursleves",11,6,
chairs could be better,9,,
"I love the office, wouldn't want to work anywere else. Maybe have more drinks and game nights",4,6,5
"1- healthier snacks, 2- bring in dinner when working late, 3- allow for lunchtime workout; and maybe a gym membership at [identifier deleted] down the street?",2,5,
fewer surveys,1,,
"drones, video games, book club, beer",6,3,5
"Supplies and tech are phenomnal, food is great, kitchen and bathrooms clean... I wouldnt change anyting.",4,,
\ No newline at end of file
+_token,_categories,_sequenceID,_reset
+string,string,int,int
```
Comment (scottpurdy) [inline data/sample_reviews_multi/sample_reviews_data_training_network.csv]: Should the _categories field be a list?

**Comment by scottpurdy** [comment]
Done with review

File: data/sample_reviews_multi/sample_reviews_data_training_network.csv (hunk 1 above)
Comment (BoltzmannBrain) [inline data/sample_reviews_multi/sample_reviews_data_training_network.csv]: yes, good catch

**Comment by scottpurdy** [comment]
:+1:

**Pull request merged**